src/Controller/DefaultController.php

```diff
@@ -92,4 +92,14 @@ public function redirectToUserSettings(AdminUrlGenerator $urlGenerator): Respons
 
         return $this->redirect($url);
     }
+
+    /**
+     * @Route("/logout", name="app_logout")
+     */
+    public function logout(): Response
+    {
+        return $this->render('default/index.html.twig', [
+            'controller_name' => 'DefaultController',
+        ]);
```

Comment:
You don't need this (cf. https://symfony.com/doc/5.x/security.html#logging-out) and why are you passing `controller_name` to the template?

Reply:
Updated!